s3/replication/doc/fdmi-to-replication-manager.plantuml

```diff
@@ -0,0 +1,41 @@
+@startuml
+
+title FDMI events for replication.\n
+
+== Queue replication jobs ==
+
+rnote over FDMI: node-1
+FDMI -> ReplicationManager: fdmi event (ev1:key1) with s3 metadata
+activate ReplicationManager
+ReplicationManager -> MOTR_KVS : GET-KV(ev1:key1)
+MOTR_KVS --> ReplicationManager : not-found
+rnote over ReplicationManager: Persist the new entry
+ReplicationManager -> MOTR_KVS : PUT-KV(ev1:key1, value{copies:0})
+MOTR_KVS --> ReplicationManager : success
+
+return status :  ack fdmi event
+
+
+rnote over FDMI: node-2
+FDMI -> ReplicationManager: fdmi event (ev1) with s3 metadata
```

Comment:
shouldn't we use (ev1:key1) as above?
or we want to keep it implicitly understood?